A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js

```diff
@@ -1631,10 +1702,19 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
         if (!failed) {
             this.bg.setAttribute("stroke", "transparent");
             this.topLine.setAttribute("stroke", "white");
-            this.bottomLine.setAttribute("stroke", "transparent");
-            this.cursorSVGShape.setAttribute("stroke", "white");
+            this.bottomLine.setAttribute("stroke", "white");
+            this.cursorSVGShape.setAttribute("stroke", "yellow");
             this.cursorSVGMainText.setAttribute("visibility", "visible");
-            this.pressureSVG.setAttribute("visibility", "visible");
+            // Code to add if visibilty issues with this part
+            
+            // if (this.pressureSVG) {
+            //     this.pressureSVGLegend.setAttribute("visibility", "visible");
+            //     this.pressureSVG.setAttribute("visibility", "visible");
+            // }
+            // if (this.STDpressureSVG) {
+            // this.STDpressureSVG.setAttribute("visibility", "visible");
+            // this.STDpressureSVGShape.setAttribute("visibility", "visible");
+            // }
```

Comment:
Please clarify this comment. If it fixes visibility issues, why is it commented out ?

Reply:
As said in the comment, the added PR for Failures in this section makes some items disappear.
 I put this code, in case there appear visisbility issues that should be resolved during testing